language/grounding/intro-grounding.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<p>"Create and use a data store in Vertex AI" → "Create and use a data store in Vertex AI Search"</p><p> </p><p> </p><p>Also change "Google Cloud ML" to "Google Cloud AI"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>